src/server/ocpp/utils/OCPPUtils.ts

```diff
@@ -2181,6 +2177,7 @@ export default class OCPPUtils {
     } else if (Utils.objectHasProperty(connector, 'amperageLimit') && connector.amperageLimit < connectorAmperageLimitMin) {
       return connectorAmperageLimitMin;
     }
+    return connectorAmperageLimitMin;
```

Comment:
The method's name is not aligned with the code:
checkAndGetConnectorAmperageLimit -> One connector

An here you multiply by the number of connectors:
`const connectorAmperageLimitMin = StaticLimitAmps.MIN_LIMIT_PER_PHASE * numberOfPhases * numberOfConnectors;`

Reply:
Fixed in https://github.com/sap-labs-france/ev-server/pull/2330